babi.py

```diff
@@ -914,7 +914,82 @@ def _get_char(stdscr: 'curses._CursesWindow') -> Key:
     return Key(wch, key, keyname)
 
 
-EditResult = enum.Enum('EditResult', 'EXIT NEXT PREV')
+EditResult = enum.Enum('EditResult', 'EXIT NEXT PREV EDIT')
+
+
+class Command:
+    def __init__(
+        self,
+        name: str,
+        description: str,
+        keybinds: List[str],
```

Comment:
these should both take `Tuple[str, ...]` instead of `List[str]` -- immutable >>> mutable when they are equivalent

This will also allow you to set a default of no aliases with `= ()` -- because tuples are immutable

Reply:
The keybinds also should be able to be changed programmatically (like the aliases) imo, just better for config reasons